web/modules/custom/menu_api/menu_api.module

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * This module helps displaying budget related information.
+ */
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Get the budget value from the configuration settings.
+ *
+ * @return int|null
+ *   The budget value if it is set; otherwise, null.
+ */
+function menu_api_get_budget() {
```

Comment:
For global functions, please create a function in a helper service.
This is so that we write more OOP code instead of procedural code.